app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -199,6 +214,23 @@ html.omni-nav-v2 {
       width: 100%;
     }
 
+    .gsib_a {
```

Comment:
Trying to keep our code as DRY as possible, lines 217-232 are repetitive of lines 386-401. You should be able to move them outside the primary nav and utility nav selectors and have them apply to both, around line 197 with the other search styles.